spec/dockerfile_parser_spec.rb

```
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+describe DockerfileParser do
+  it 'parses simple from statements' do
```

RuboCop (1.75.5)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)